bin/burden

```diff
@@ -1736,21 +1804,28 @@ create_ansible_options()
 			done
 			popd > /dev/null
 		fi
-		if [[ "$gl_ssh_key_file" == "" ]]; then
-			kick_off.sh -t $gl_top_dir -d $run_dir -f $gl_sys_file -a $gl_create_attempts -S $gl_spot_recover -c $gl_test_def_dir $rm_dirs | tee ${run_dir}/ansible_log &
-		else
-			kick_off.sh -t $gl_top_dir -d $run_dir -f $gl_sys_file -a $gl_create_attempts -S $gl_spot_recover -s $gl_ssh_key_file -c $gl_test_def_dir $rm_dirs | tee ${run_dir}/ansible_log &
-		fi
-		pids[${pindex}]=$!
-		let "pindex=$pindex+1"
-		if [[ $pindex -eq $gl_max_systems ]]; then
-			for pid in ${pids[*]}; do
-				wait $pid
-			done
-			pindex=0
+		if [ $gl_preflight -eq 0 ]; then
+			# DJV
+			#cleanup_and_exit "No execute" 1
+			if [[ "$gl_ssh_key_file" == "" ]]; then
+				kick_off.sh -t $gl_top_dir -d $run_dir -f $gl_sys_file -a $gl_create_attempts -S $gl_spot_recover -c $gl_test_def_dir $rm_dirs | tee ${run_dir}/ansible_log &
+			else
+				kick_off.sh -t $gl_top_dir -d $run_dir -f $gl_sys_file -a $gl_create_attempts -S $gl_spot_recover -s $gl_ssh_key_file -c $gl_test_def_dir $rm_dirs | tee ${run_dir}/ansible_log &
+			fi
```

Comment:
Consider using the if to build different options between these 2 cases in a variable.

Less copy-paste errors later

Reply:
Done.